app/src/main/java/com/github/gogiffy/adapters/GifAdapter.java

```diff
@@ -95,6 +95,8 @@ public void clearAll() {
         notifyDataSetChanged();
     }
 
+    public void TestHere(){}
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)